File: #04/q02 - frequencia/frequencia.hs
frequencia :: (Num a, Eq t) => t -> [t] -> a
frequencia a [] = 0
frequencia a (x:xs) = 
    if a == x then 1 + frequencia a xs
    else 0 + frequencia a xs




main :: IO ()
main = do
    a <- readLn :: IO Int
    b <- readLn :: IO [Int]
    print $ frequencia a b

File: #04/q02 - frequencia/frequencia.js
frequencia = (x, arr) => arr.length == 0? 0
    : (x == arr.pop()? 1 + frequencia(x, arr): 0 + frequencia(x,arr))

console.log(frequencia(1,[4,4]))
console.log(frequencia(5,[4,5,2,1,5,5,9]))
console.log(frequencia(1,[]))